### PR TITLE
Feature #3016: Add git version into oned -v

### PR DESCRIPTION
**Pull request by tinova**
To be squashed and cherry-picked into one-5.8

### CODE REVIEW

File: SConstruct [hunk 1]
```diff
@@ -269,8 +280,7 @@ build_scripts=[
     'share/rubygems/SConstruct',
     'src/im_mad/collectd/SConstruct',
     'src/client/SConstruct',
-    'src/docker_machine/SConstruct',
-    svncterm_path
```
Comment (rsmontero) [inline SConstruct]: needed

File: include/Nebula.h [hunk 2]
```diff
@@ -362,7 +362,12 @@ class Nebula
      */
     static string code_version()
     {
-        return "5.9.80"; // bump version
+       ostringstream os;
+
+       // bump version
+       os << "5.9.80 (" << GITVERSION << ")";
```
Comment (rsmontero) [inline include/Nebula.h]: bump version aware? (i.e. pattern still applies or sed line)

File: SConstruct (hunk 1 above)
Comment (tinova) [reply SConstruct]: true, my fault since i remove it for mac compilation

**Comment by tinova** [comment]
Closing, changing the git version return

**Pull request closed without merging**